### DISCUSSION TITLE
Nested ports

### DISCUSSION THREAD

**Discussion by aviskiba** [eclipse-syson/syson#739] [Q&A] [answered]
Hello SysON Team,

1st of all I have to say its a very cool tool you are making. Tanks for that.

Then I have a question: Is there a way to display nested ports of ports of a part in Interconnection view?
I tried it directly or via a prot def:
![image](https://github.com/user-attachments/assets/4070ae10-0f16-4622-af56-d31d7978373f)

But I just get:

![image](https://github.com/user-attachments/assets/d7c72dfd-e15c-4697-a1a1-85c86b93046b)

**Comment by AxelRICHARD** [comment]
Hello,
Thank you for your kind comment!
SysON is based on [Sirius Web](https://github.com/eclipse-sirius/sirius-web), and Sirius Web does not allow yet to define ports in ports.
So unfortunately, this is not possible yet in SysON.
But we know that this is something that will need to be addressed in the future, and will do our best to have that in SysON.
Best regards,

**Comment by aviskiba** [comment]
Hello,
so you we will have to wait until this is supported by Sirius Web?
Ist this featur on the Sirius Web road map?
And if so, can you share a link to this information.
Best regards

**Reply by AxelRICHARD** [reply, marked as answer]
Hello,
Yes SysON will have to wait until this is supported by Sirius Web.
No, this feature is not on the roadmap yet, but the Sirius Web team is aware of the need and we discuss about that with them.
Best regards,
